lib/google_maps_service/apis/roads.rb

```diff
@@ -63,8 +63,9 @@ def snap_to_roads(path, interpolate: false)
     #         by the snap_to_roads function. You can pass up to 100 Place IDs.
     #
     # @return [Array] Array of speed limits.
-    def speed_limits(place_ids)
+    def speed_limits(place_ids, speed_unit='KPH')
```

Comment:
I think your example in the PR description can be put in the method example here, so it will be in documentation also :)